builder/hyperv/common/driver_ps_4.go

```diff
@@ -291,33 +291,44 @@ func (d *HypervPS4Driver) verifyPSHypervModule() error {
 		return err
 	}
 
-	res := strings.TrimSpace(cmdOut)
-
-	if res == "False" {
+	if powershell.IsFalse(res) {
 		err := fmt.Errorf("%s", "PS Hyper-V module is not loaded. Make sure Hyper-V feature is on.")
 		return err
 	}
 
 	return nil
 }
 
-func (d *HypervPS4Driver) verifyHypervPermissions() error {
-
-	log.Printf("Enter method: %s", "verifyHypervPermissions")
-
+func (d *HypervPS4Driver) isCurrentUserAHyperVAdministrator() (bool, error) {
 	//SID:S-1-5-32-578 = 'BUILTIN\Hyper-V Administrators'
 	//https://support.microsoft.com/en-us/help/243330/well-known-security-identifiers-in-windows-operating-systems
-	hypervAdminCmd := "([Security.Principal.WindowsPrincipal][Security.Principal.WindowsIdentity]::GetCurrent()).IsInRole('S-1-5-32-578')"
+
+	var script = `
+$identity = [System.Security.Principal.WindowsIdentity]::GetCurrent()
```

Comment:
Will this work in other languages?  I remember there was an issue when checking for whether someone was an administrator in the normal windows OS.